### ISSUE TITLE
BPE n'est plus disponible à cet url

### ISSUE THREAD

**Issue by linogaliana** [InseeFrLab/DoReMIFaSol#65]
Il semblerait que la BPE ait changé d'URL

```r
doremifasol::telechargerDonnees("BPE_ENS")
```

```r
essai de l'URL 'https://www.insee.fr/fr/statistiques/fichier/3568629/bpe19_ensemble_csv.zip'
Error in download.file(url = caract$lien, destfile = nomFichier) : 
  impossible d'ouvrir l'URL 'https://www.insee.fr/fr/statistiques/fichier/3568629/bpe19_ensemble_csv.zip'
De plus : Warning message:
In download.file(url = caract$lien, destfile = nomFichier) :
  cannot open URL 'https://www.insee.fr/fr/statistiques/fichier/3568629/bpe19_ensemble_csv.zip': HTTP status was '500 '
```

**Comment by py-b** [comment]
Effectivement, cette page semble dynamique, la BPE 2020 a remplacé la BPE 2019 à cette url : https://www.insee.fr/fr/statistiques/3568629/

Le fichier s'appelle maintenant `bpe20_ensemble_csv.zip`.

**Comment by pierre-lamarche** [comment]
Bien reçu, normalement ce genre de travaux est scripté là : https://github.com/InseeFrLab/DoReMIFaSol/blob/master/data-raw/archive/liste_donnees.R
La bonne réponse est je pense à terme d'automatiser ce travail de mise à jour de la base de données.

**Comment by linogaliana** [comment]
Dans le cadre de [pynsee](https://github.com/InseeFrLab/Py-Insee-Data/pull/19), je voulais écrire un script qui check de manière automatique l'ensemble des clés du JSON. Ce script pourrait tourner à intervalles réguliers avec `cron`. 

Cela donne un indicateur régulier des clés à mettre à jour

**Comment by py-b** [comment]
J'ai corrigé les noms de fichiers : 1b6397db07b5fe1022ccb50b25eef1bfb628ad23
Les métadonnées restent pour l'instant celles de la BPE 2019.

**Comment by py-b** [comment]
Remplacé 2020 par 2021.
